packages/babel-helper-hoist-variables/src/index.ts

```diff
@@ -40,6 +40,12 @@ const visitor = {
 
     for (const declar of declarations) {
       firstId = declar.node.id;
+      if (
+        declar.parentPath.parentPath.isBlockStatement() &&
+        declar.parent.kind !== "var"
```

Comment:
Nit: we can hoist this check outside of the `for` loop:
```js
const needsRename = path.node.kind !== "var" && path.parentPath.isBlockStatement()
```

Reply:
Actually, I think we can just edit the `Scope` visitor above to check `kind === "let" || kind === "const"`.

Reply:
Okay, I will change that.